### ISSUE TITLE
Fails to insert mention if no suggestion is specified.

### ISSUE THREAD

**Issue by jameskraus** [withspectrum/slate-mentions#10]
If no suggestion is specified, [this block of code fails](https://github.com/withspectrum/slate-mentions/blob/44c76aea64fff1d76c30bf137ddd8cee07cf95c3/src/index.js#L82-L98) to insert a suggestion.

              var selected = Math.abs(editor.state[_constants.SELECTED_MENTION_INDEX_KEY]); // selected = 0
              var text = suggestions[selected % suggestions.length || 0]; // text = undefined
              var at = (0, _utils.nearestAt)(state.endText.text, state.selection.startOffset);
              var mentionLength = state.selection.startOffset - at - 1;

              // Remove the current, incomplete text and replace it with the selected suggestion
              return state.transform().deleteBackward(mentionLength).insertText(text).removeMark('mention').insertText(' ').focus().apply(); // fails to insert `undefined` text

I get this error: `Cannot read property 'length' of undefined`

 To reproduce, just enter an @mention that doesn't exist in the example and it will throw the error.

The two normal ways of handling this are to either create an ad-hoc mention or to not allow the user to create the mention at all. Considering the generality of this library, it seems appropriate to handle both cases?

**Comment by mxstbr** [comment]
Yep yep. This is very alpha, so lots of functionality like that is still missing :wink: Would appreciate PRs!
